src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/auto_crafters/AbstractAutoCrafter.java

```diff
@@ -416,21 +417,20 @@ public void register(@Nonnull SlimefunAddon addon) {
         if (getCapacity() <= 0) {
             warn("The capacity has not been configured correctly. The Item was disabled.");
             warn("Make sure to call '" + getClass().getSimpleName() + "#setEnergyCapacity(...)' before registering!");
+            return;
```

Comment:
No

Reply:
but why? now you have two ifs checking the same thing?

Reply:
We want to send a warning if one value was misconfigured.
But we still want to send both warnings if both values were misconfigured.

That's why a `return;` here is very much unwanted since it would swallow the second warning message.
Hope that clears it up for you